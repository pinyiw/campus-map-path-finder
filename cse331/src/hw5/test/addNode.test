# Create a graph
CreateGraph graph1

#Add nodes
AddNode graph1 a
AddNode graph1 b
AddNode graph1 c