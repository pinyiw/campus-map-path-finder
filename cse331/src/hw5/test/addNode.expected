# Create a graph
created graph graph1

#Add nodes
added node a to graph1
added node b to graph1
added node c to graph1